jobs/rocoto/prepoceanobs.sh

```diff
@@ -12,6 +12,11 @@ status=$?
 export job="prepoceanobs"
 export jobid="${job}.$$"
 
+###############################################################
+# setup python path for class defs and utils
+
+export PYTHONPATH=${HOMEgfs}/ush:${PYTHONPATH}
```

Comment:
```suggestion
PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${HOMEgfs}/ush"
```
If `PYTHONPATH` does not exist in the environment, the path will end with `:`